### DISCUSSION TITLE
How do I rerun the build script so that cord can see it?

### DISCUSSION THREAD

**Discussion by DrTeagle** [vyfor/cord.nvim#113] [Q&A] [answered]
probably a noob question but I am in a scenario where I get the error that I need to rerun the build script, however whenver I do cargo build it doesnt seem to work?

**Comment by vyfor** [comment, marked as answer]
Hey, you need to run the `build.sh` script found in the cord directory

**Reply by DrTeagle** [reply]
do you mean build, and that worked, ty.
